libs/utils/utils.js

```diff
@@ -92,16 +92,16 @@ const AUTO_BLOCKS = [
   { gist: 'https://gist.github.com' },
   { caas: '/tools/caas' },
   { faas: '/tools/faas' },
-  { fragment: '/fragments/' },
+  { fragment: '/fragments/', voidStyles: true },
```

Comment:
To save bytes, I would advocate for `styles: false` or `stylesOff`. Void would be new vocabulary for engineering and GWP.

Reply:
Sure, thanks, sounds good, I was actually mulling over this as well